pkgs/tools/filesystems/bindfs/default.nix

```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ fuse ];
+  buildInputs = [ (if stdenv.isDarwin then osxfuse else fuse) ];
```

Comment:
I would appreciate it if this was reformatted as such:
```
buildInputs = if stdenv.isDarwin
              then [ osxfuse ]
              else [ fuse ];
```

Reply:
I think it is fine as is.

Reply:
I'm the active maintainer currently, I'd like to keep it within my style preferences

Reply:
Should Nix indentation depend on the length of identifiers? I'm fine with either 😄.

Reply:
Reformatted. I use https://github.com/nix-community/nixpkgs-fmt to format code on save and it produce quite decent results, at some point it may be accepted as a default formatter for that repo (that's stated as its goal in the readme).
So probably it's better not to deviate much from the formatting produced by nixpkgs-fmt to decrease the size of the diff in future and don't have endless wars on format preferences.